_implementors/servers.md

```diff
@@ -74,6 +74,7 @@ index: 1
 | [MSBuild](https://github.com/Microsoft/msbuild) | [Adam Friedman](https://github.com/tintoy) | [msbuild-project-tools-vscode](https://github.com/tintoy/msbuild-project-tools-vscode) | TypeScript |
 | Nim | [Peter Munch-Ellingsen](https://github.com/PMunch) | [nimlsp](https://github.com/PMunch/nimlsp) | Nim |
 | OCaml/Reason | [freebroccolo](https://github.com/freebroccolo) | [ocaml-language-server](https://github.com/freebroccolo/ocaml-language-server) | TypeScript |
+| OCaml/Reason | [OCaml Labs](http://ocamllabs.io) | [ocamllsp](https://github.com/ocaml/ocaml-lsp) | OCaml |

```

Comment:
Should the entry above this be deleted? 🤔

Reply:
Actually, I guess they aren't the same project? I thought the older one got moved somewhere else (since the link no longer works) but I guess they're not the same thing as the old one was written in TypeScript and this other one is written in OCaml?

Reply:
It looks like it was moved to https://github.com/ocaml-lsp/ocaml-language-server, and yes they're different projects.